doc/release-notes/0.27/0.27.md

```diff
@@ -78,6 +78,13 @@ The following table covers notable changes in v0.27.0. Further information about
 The <tt>_JAVA_OPTIONS</tt> environment variable (with different behavior) is added for compatibility.</td>
 </tr>
 
+<tr><td valign="top">
+<a href="https://github.com/eclipse-openj9/openj9/pull/12501">#12501</a></td>
+<td valign="top">Increase OS stackspace on PPC64 platforms</td>
```

Comment:
Can we please spell out "OS" and should we use "stack size" as in version0.27.md rather than "stackspace"?  `Increase operating system stack size on PPC64 platforms`